bucket/gitlab-runner.json

```diff
@@ -26,6 +26,9 @@
             "32bit": {
                 "url": "https://gitlab-runner-downloads.s3.amazonaws.com/v$version/binaries/gitlab-runner-windows-386.exe#/gitlab-runner.exe"
             }
+        },
+        "hash": {
+            "url": "https://gitlab-runner-downloads.s3.amazonaws.com/v$version/release.sha256"
```

Comment:
```suggestion
            "url": "$baseurl/release.sha256"
```

Reply:
Please notice additional `binaries` in url.